kong/conf_loader.lua

```diff
@@ -222,6 +223,11 @@ local function check_and_infer(conf)
       "DCAwareRoundRobin policy is in use"
     end
 
+    if conf.cassandra_refresh_frequency < 0 then
```

Comment:
What if `cassandra_refresh_frequency` is not set?

Reply:
This is why kong has default values, what we can do is set here the default value so we dont fail